### DISCUSSION TITLE
The running time of Yalmip  becomes longer and longer as the number of cycles increases

### DISCUSSION THREAD

**Discussion by Zierhappy** [yalmip/YALMIP#1002] [Problems using YALMIP]
I used Yalmip for integer programming problems, but it only running fast in first circle. I've already used "yalmip('clear')" after "for....."  in my programming. However, it does not work effectively jast like first circle. It is note that the model is written in another file,  which is called each time it is executed. How to release all variables in Yalmip after each execution?

**Comment by johanlofberg** [comment]
Still sounds like your logic is flawed so you keep racking up stuff internally without clearing.
